en/clubs/desc/body.md

```diff
@@ -1,3 +1,43 @@
-# The Esport Community
+# DESC
+
+DESC (*Datas E-Sports Community*) is the CS chapter's gaming club. Our mission is to spread the joy of gaming to the CS chapter. We are an open club: if you're a member in the chapter, you're a member of DESC. DESC is headed by the Desctop, elected at Val-SM each year.
+
+## What we do
+
+We organize a slew of activities related to e-sports and gaming in general. For example, we...
+
+* attend Dreamhack Winter yearly (sponsored by the chapter),
+* organize tournaments with prizes in META,
+* watch e-sports events in META,
+* hold on- and offline gaming nights,
+* speed run Dark Souls,
+* convince fellow students to show off their VR gear,
+* get wrecked in Overwatch,
+* encourage members of the chapter to play more video games with eachother.
+
+Any other time we're hanging out in the Discord server.
+
+## Discord
+
+Join our Discord server!
+
+<a href="https://discord.gg/xwjCxXkmFM">
+    <img 
+         src="http://static.datasektionen.se/namnder/desc/discord-logo.png" 
```

Comment:
Fungerar ej, kommer ej laddas in då static ligger bakom auth. Gjorde den publik, så¨nu bör det fungera

Reply:
Om man däremot laddar upp en fil som publik fungerar det. Tog bort och laddade upp igen, så nu bör det fungera